### PR TITLE
Update Ui and btn also ,create a dark mode theme who depend on  device

### PR DESCRIPTION
**Pull request by Maan-Sharma**
If you want to update any specific part please suggest me.

### CODE REVIEW

**Comment by ashut0sh75** [comment]
Will you please share the images or a video of what you have updated. Thank you.

**Comment by Maan-Sharma** [comment]
WhatsApp Image 2023-10-08 at 9 34 55 PM
The Dark Mode Ui

**Comment by Maan-Sharma** [comment]
WhatsApp Image 2023-10-08 at 9 34 56 PM (1)
The  Light Ui

**Comment by Maan-Sharma** [comment]
WhatsApp Image 2023-10-08 at 9 34 56 PM
The Fancy Toast

File: .idea/gradle.xml
Comment (lingarajsankaravelu) [inline .idea/gradle.xml:1]: you can drop the changes under .idea folder

File: app/src/main/java/com/example/cartoon/MainActivity.kt [hunk 1]
```diff
@@ -77,7 +89,10 @@ class MainActivity : ComponentActivity() {
 
         // Check if no image is selected
         if (drawable == null) {
-            Toast.makeText(this, "No image selected", Toast.LENGTH_SHORT).show()
+            FancyToast.makeText(
+                this@MainActivity,
+                "No image selected",FancyToast.LENGTH_LONG,
```
Comment (lingarajsankaravelu) [inline app/src/main/java/com/example/cartoon/MainActivity.kt:94]: You can define all the messages which is used in this file under string.xml file and read it back  using context. This improves code quality and makes it easier for the next code review on message changes.

File: app/src/main/java/com/example/cartoon/MainActivity.kt [hunk 2]
```diff
@@ -77,7 +89,10 @@ class MainActivity : ComponentActivity() {
 
         // Check if no image is selected
         if (drawable == null) {
-            Toast.makeText(this, "No image selected", Toast.LENGTH_SHORT).show()
+            FancyToast.makeText(
```
Comment (lingarajsankaravelu) [inline app/src/main/java/com/example/cartoon/MainActivity.kt:92]: See whether  it is possible to move FancyToast to a helper function and pass just the required argument.

**Comment by ashut0sh75** [comment]
Thank you for your contribution in our Project.

**Review by lingarajsankaravelu — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/cartoon/MainActivity.kt (hunk 2 above)
Comment (ashut0sh75) [reply app/src/main/java/com/example/cartoon/MainActivity.kt:92]: Would you like to do it? You can open an issue for this and we will assign it to you. Thank you for suggestion @lingarajsankaravelu.

Comment (lingarajsankaravelu) [reply app/src/main/java/com/example/cartoon/MainActivity.kt:92]: why not. @ashut0sh75 . I think you approved the PR in between my review.

Comment (ashut0sh75) [reply app/src/main/java/com/example/cartoon/MainActivity.kt:92]: Yes that was the case.

**Pull request merged**